example/lib/ios_popover/_ios_popover_menu.dart

```diff
@@ -0,0 +1,222 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+class IosPopoverMenu extends StatelessWidget {
```

Comment:
Please provide a Dart Doc that introduces the reader to the important aspects of this widget.

Reply:
Dart Doc added.